aesthetic_ants/penguin/enemy.py

```diff
@@ -0,0 +1,13 @@
+import pyglet
+import random as rand
+
+
+class Enemy(pyglet.sprite.Sprite):
+
+    def __init__(self, enemy_image):
+        pyglet.sprite.Sprite.__init__(self, enemy_image)
```

Comment:
Use `super()` instead of having to explicitly call the parent class' `__init__`
```suggestion
        super().__init__(enemy_image)
```